docs/platforms/javascript/common/configuration/integrations/index.mdx

```diff
@@ -50,7 +50,19 @@ For all other cases, we recommend you use the `integrations` option.
 
 ## Lazy Loading Integrations
 
-You can lazy load pluggable integrations via `Sentry.lazyLoadIntegration()`. This will try to load the integration from the CDN, allowing you to add an integration without increasing the initial bundle size.
+Lazy loading lets you add pluggable integrations without increasing the initial bundle size. You can do this in two ways:
+
+### 1. Dynamic Import (recommended)
+
+You can add the integration with a dynamic import using `import()`. This method loads the integration from the npm package. To avoid running into issues with `import()`, you should use a bundler that supports dynamic imports. If you're using a tool like Vite for your project, the bundling process is probably already set up.
```

Comment:
```suggestion
You can add the integration with a dynamic import using `await import()`. This method loads the integration from the npm package. To avoid running into issues with `import()`, you should use a bundler that supports dynamic imports. If you're using a tool like Vite for your project, the bundling process is probably already set up.
```

maybe slightly clearer?